docs/content/preview/migrate/reference/yb-voyager-cli.md

```diff
@@ -370,15 +566,15 @@ Path to set `$ORACLE_HOME` environment variable. `tnsnames.ora` is found in `$OR
 
 ### --use-orafce
 
-Enable using orafce extension in export schema.
+Enable using orafce extension in export schema. Applicable for Oracle only.
```

Comment:
could use a little more info here - what's it used for, where does the extension need to be installed, the source and the target? just the source? what if you turn this flag on but haven't installed the extension?

'Oracle migrations only.'

Reply:
@sanyamsinghal: Could you please provide some details about this ?